app/controllers/products_controller.rb

```diff
@@ -0,0 +1,56 @@
+class ProductsController < ApplicationController
+
+
+  def new 
+    @product = Product.new
+  end
+
+  def create
+    @product = Product.new(product_create_params[:product])
+    if @product.save
+      redirect_to vendor_path(@product.vendor_id)
+    else
+    render :new
+    end
+  end
+
+  def update
+    product = Product.find(params[:id])
+    # task = Task.find(params[:id])
```

Comment:
See comment above.  It's common to remove any unnecessary comments before submitting a PR.